docs/custom-flows/embedded-email-links.mdx

```diff
@@ -1,46 +1,105 @@
 ---
-title: Embeddable email links
-description: Learn how to build custom email link flows to increase user engagement and reduce drop off in transactional emails, SMS's, and anywhere else you can imagine.
+description: Learn how to build custom embeddable email link sign-in flows to increase user engagement and reduce drop off in transactional emails, SMS's, and more.
 ---
 
-# Embeddable email links
+# Embeddable email links with sign-in tokens
 
-An "email link" is a link, that when pressed, will automatically authenticate your user, so that they can quickly perform some action on your site with less friction than if they had to sign in manually.
+An "email link" is a link that, when visited, will automatically authenticate your user so that they can perform some action on your site with less friction than if they had to sign in manually. You can create email links with Clerk by generating a sign-in token, which you can detect and take action on in your frontend.
 
 Common use cases include:
 
-- Welcome emails, when users are added off a waitlist
-- Promotional emails for users
-- Recovering abandoned carts
-- Surveys or questionnaires
+- Welcome emails when users are added off a waitlist.
+- Promotional emails for users.
+- Recovering abandoned carts.
+- Surveys or questionnaires.
```

Comment:
This is a list, not sentence-finishers. If these are fragments that were finishing the initial sentence, then you'd add periods!

```suggestion
- Welcome emails when users are added off a waitlist
- Promotional emails for users
- Recovering abandoned carts
- Surveys or questionnaires
```